app/views/pages/index.html.erb

```diff
@@ -1,4 +1,23 @@
 <div class="container text-center">
   <h1>Pages#index</h1>
   <p>Find me in app/views/pages/index.html.erb</p>
+  <% if current_user %>
```

Comment:
@jacquesporveau @YarekTenko Yarek is correct. current_user is a method defined in his sessions_helper and as such it will included in all view contexts